src/gfn/examples/box_utils.py

```diff
@@ -803,9 +830,8 @@ def to_probability_distribution(
             marker="x",
         )
 
-    delta = 0.2  # TODO: this does not work with 0.1, why? Triage with Salem.
+    delta = 0.1  # TODO: this does not work with 0.1, why? Triage with Salem. ? Salem: Doesn't it ???
```

Comment:
sorry this is an outdated comment and is wrong -- it does work, I had a bug elsewhere when I wrote this.